charts/hub-ui/templates/environment.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "hub-ui.fullname" . }}
+data:
+  KRATOS_ENDPOINT: {{ .Values.environment.kratosPublicEndpoint }}
+  PORT: "{{ .Values.environment.port }}"
```

few more env to handle.

https://github.com/holaplex/hub/blob/main/k8s/deployment.yaml#L21

```suggestion
  {{- with .Values.environment }}
  KRATOS_ENDPOINT: {{ .kratosPublicEndpoint }}
  PORT: "{{  .port }}"
  {{- end }}
```

with scoping is kinda nice

---

I was kinda doubting to add them at all, because those get baked into the container when building.
what do you think? should i remove them alltogether?

https://github.com/holaplex/hub/blob/main/.github/workflows/main.yml#L38-L41

---

done!